parts/kubernetesmasteraddons-calico-daemonset.yaml

```diff
@@ -217,8 +217,10 @@ spec:
       hostNetwork: true
       serviceAccountName: canal
       tolerations:
-        - key: node-role.kubernetes.io/master
-          effect: NoSchedule
+        - key: "node-role.kubernetes.io/master"
+          operator: "Equal"
+          value: "true"
```

Comment:
does it work without quotes across the board? Should we be explicitly quoting strings? These files appear to have a mix